registration/machines_api_reg.lua

```diff
@@ -202,9 +206,9 @@ logistica.register_requester("Bulk Request Inserter\nInserts up to 64 items at a
 -- Reservoirs
 --------------------------------
 
-logistica.register_reservoir("lava", "Lava", "bucket:bucket_lava", "default_lava.png", 8)
-logistica.register_reservoir("water", "Water", "bucket:bucket_water", "default_water.png")
-logistica.register_reservoir("river_water", "River Water", "bucket:bucket_river_water", "default_river_water.png")
+logistica.register_reservoir("lava", "Lava", itemstrings.lava_bucket, "default_lava.png", 8)
+logistica.register_reservoir("water", "Water", itemstrings.water_bucket, "default_water.png")
+logistica.register_reservoir("river_water", "River Water", itemstrings.river_water_bucket, "default_river_water.png")
```

Comment:
I think the textures (the 4th parameter) need to be updated, maybe need another table in compat_mcl like `logistica.base_textures` or something - because I doubt MCL also uses `default_water.png` right?

Reply:
Right. I haven't gotten that far yet, since the only water/lava textures MCL has are animated (although there's probably a texture modifier that grabs a specific part of the texture, but I just haven't checked yet).

Reply:
Yeah, I downloaded MCL2, using the animated texture doesn't really work unfortunately
![image](https://github.com/ZenonSeth/logistica/assets/118483769/f3ff0bba-aeb1-4a1e-ba3f-c98febecf127)

I don't think there's a crop modifier to create textures unfortunately - if that becomes an issue I can include some MCL specific textures in the modpack that match the default MCL looks. I know that isn't idea if there's texture packs, but these textures aren't shown that much, and as long as they are indicative of the liquid type, it should be fine.

Reply:
I found a solution: `^[sheet:1x16:0,0`